<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>remove_stock_window</class>
 <widget class="QDialog" name="remove_stock_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Remove Stock&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="item_dpci_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Item DPCI:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Item_location_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Item Location:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Item Dept:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="remove_amount_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Remove Amount:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="remove_dpci_edit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="remove_loc_edit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="remove_amount_edit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_remove_button">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>450</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm Remove</string>
   </property>
  </widget>
  <widget class="QComboBox" name="remove_dept_input">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OFCE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>STAT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TOY</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BATH</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BEDS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DOMS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BABY</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PLUG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PLAS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FURN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BPLS</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
